### PR TITLE
Mm 09 rover photo viewer

### PR DESCRIPTION
**Pull request by clarkanastasia**
Created Photo Viewer component which includes: 
1. heading with page name
2. subheading with instructions for user
3. three buttons allowing user to select the rover
4. paragraph displaying which rover is currently selected
5. large div to display the image selected by user from thumbnails grid
6. paragraph with date the image was taken
7. grid of thumbnail images from selected rover

On first render, the component will display images from the "spirit" rover and use its first image as the default for the cover image.
When the user clicks on an image thumbnail, the cover image is replaced with the selected image and the selected image is highlighted with a border in the thumbnail grid. 
Screenshot 2024-02-14 180205
Screenshot 2024-02-14 180106

### CODE REVIEW

File: src/components/App/index.tsx [hunk 1]
```diff
@@ -1,8 +1,9 @@
 import React from "react";
+import "./styles.scss";
+import PhotoViewer from "../PhotoViewer/PhotoViewer";
```
Comment (Louisasa) [inline src/components/App/index.tsx]: can we remove this import now it isn't being used here? thanks :)

File: src/components/PhotoViewer/PhotoViewer.tsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+interface RoverData {
+  photos: Photo[];
+}
+
+const PhotoViewer: React.FC = () => {
+  const [roverData, setRoverData] = useState<RoverData>();
+  const [selectedRover, setSelectedRover] = useState<string>("spirit");
+  const [coverImageUrl, setCoverImageUrl] = useState<string>("");
+  const [date, setDate] = useState<string>();
+  const URL = `https://api.nasa.gov/mars-photos/api/v1/rovers/${selectedRover}/photos?api_key=${process.env.REACT_APP_API_KEY}&sol=100`;
```
Comment (Louisasa) [inline src/components/PhotoViewer/PhotoViewer.tsx:20]: nice work here! would you mind adding instructions to the README about this? it doesn't need to be too in depth but mostly the stating variable name and file (this is mostly for if new people would join the project but also good for when the others pull this change need to update their env file)

**Review by Louisasa — COMMENTED** [review]
looking good here! prettier is struggling to run because of a new install you did, I didn't notice it being used is it definitely necessary? if it is, can you please run `pnpm import` pnpm is another package manager like npm and prettier runs off of pnpm so it just wants a copy of the new install to be able to run :) thanks!

File: src/components/App/index.tsx (hunk 1 above)
Comment (clarkanastasia) [reply src/components/App/index.tsx]: that's done!

File: src/components/PhotoViewer/PhotoViewer.tsx (hunk 2 above)
Comment (clarkanastasia) [reply src/components/PhotoViewer/PhotoViewer.tsx:20]: updated README with an API key section

**Review by Louisasa — APPROVED** [review]

**Pull request merged**